garminconnect/__init__.py

```
@@ -618,6 +625,90 @@ def get_max_metrics(self, cdate: str) -> Dict[str, Any]:
+
+                # This is not a typo. The Garmin dictionary has a typo as of 2025-07-08, refering to it as "hearRate"
+                elif entry['hearRate'] is not None:
+                    speed_and_heart_rate_dict["heartRate"] = entry["hearRate"] # Fix Garmin's typo
+
+                # Doesn't exist for me but adding it just in case.  We'll check for each entry
+                if entry['heartRateCycling'] is not None:
+                    speed_and_heart_rate_dict["heartRate"] = entry["heartRateCycling"]
+
+            latest_dict = {
+                "speed_and_heart_rate": speed_and_heart_rate_dict,
+                "power": power_dict
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)